src/Callbacks.php

```diff
@@ -36,6 +37,29 @@ public function setOnSend($onSend)
     {
         $this->onSend = $onSend;
     }
+ 
+     /**
+     * Call the callback function
+     * @param string $from
+     * @param string $messageId
+     * @param string $status
+     * @param string $timestamp
```

Comment:
```suggestion
     * @param int $timestamp
```

Reply:
Agreed, string is the wrong thing to do here, quick copy/paste problem. However, this can get quite big, as it has a millisecond granularity (javascript timestamap) instead of the usual unix timestamp. While php should be fine, this can get problematic with SQL databases, so it should be documented.

Reply:
Indeed. I'll let you document it in your future PR.